src/Faker/Provider/es_MX/Person.php

```diff
@@ -0,0 +1,337 @@
+
+
+    public function lastName()
+    {
+        return static::randomElement(static::$lastName).' '.static::randomElement(static::$lastName);
+    }
+
+    public function lastNameMother()
+    {
+        return static::randomElement(static::$lastNames);
+    }
+
+    public function lastNameFather()
+    {
+        return static::randomElement(static::$lastNames);
+    }
+    /**
+     * generates CURP
+     *
+     * @param      string  $firstName       First name
+     * @param      string  $lastNameFather  Fathers last name
+     * @param      string  $lastNameMother  Mothers last name
+     * @param      \DateTime  $birthDate    Birth date
+     * @param      string  $gender          Gender
+     * @param      string  $state           State of birth
+     *
+     * @return     string  generated curp
+     *
+     * @link       https://github.com/hectorip/RFC-CURP-Mexico/blob/master/src/mxk.js ported from this
+     * @link       https://en.wikipedia.org/wiki/Unique_Population_Registry_Code
+     */
+    public static function curp($firstName = null, $lastNameFather = null, $lastNameMother = null, $birthDate = null, $gender = null, $state = null)
```

Comment:
CS around this method: please leave space before and after ternary condition operators. `a ? b : c`